charts/atlas-cluster/templates/atlas-project.yaml

```diff
@@ -12,13 +12,15 @@ metadata:
   {{- end }}
 spec:
   name: {{ .Values.project.atlasProjectName }}
+{{- if ( not .Values.mongodb-atlas-operator.clusterWide) }}
   connectionSecretRef:
```

Comment:
if the secret is not specified, the operator will use the secret in the operator namespace.